src/vs/workbench/parts/debug/browser/debugActions.ts

```diff
@@ -133,7 +133,11 @@ export class StartAction extends AbstractDebugAction {
 		this.toDispose.push(this.contextService.onDidChangeWorkbenchState(() => this.updateEnablement()));
 	}
 
-	public run(): Thenable<any> {
+	public run(config?: string | IConfig): Thenable<any> {
```

Comment:
@isidorn fyi

Reply:
@RMacfarlane not the nicest. But fine for me. Why do you have `string | IConfig`, when you always pass an `IConfig` as far as I understand?

Reply:
Also please add a comment for the use case here. So I do not accidently delete this code later

Reply:
I updated it to be only `IConfig` and added a note